performance-tests/perf-tests.sh

```diff
@@ -163,7 +251,15 @@ ON_SHUTDOWN+=("rm -rf ${TMP}")
 
 # Bring up Kafka
 ON_SHUTDOWN+=("runDockerCompose down")
-runDockerCompose pull
+
+# This doesn't work with podman if I want to push to the local repository
+#runDockerCompose pull
```

Comment:
```suggestion
[[ -n ${PULL_CONTAINERS} ]] && runDockerCompose pull
```

We would then make CI export `PULL_CONTAINERS`